pandas/io/tests/test_network.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+
+"""
+Tests parsers ability to read and parse non-local files
```

Comment:
so some of the files are not in the ``pandas/io/tests/parser/*`` namespace e.g.
``pandas/io/tests/test_network.py`` as an example.

```
[jreback-~/pandas] git diff master --stat
 pandas/io/tests/parser/__init__.py        |    0
 pandas/io/tests/parser/compression.py     |  155 +++++
 pandas/io/tests/parser/parse_dates.py     |  468 +++++++++++++++
 pandas/io/tests/parser/test_parsers.py    | 3342 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 pandas/io/tests/parser/test_textreader.py |  402 +++++++++++++
 pandas/io/tests/test_cparser.py           |  401 -------------
 pandas/io/tests/test_network.py           |  195 +++++++
 pandas/io/tests/test_parsers.py           | 5055 --------------------------------------------------------------------------------------------------------------------------------------------------------------
 pandas/io/tests/test_read_fwf.py          |  337 +++++++++++
 pandas/io/tests/test_unsupported.py       |   88 +++
 pandas/tests/test_lib.py                  |   74 ++-
```